bciers/apps/administration/app/components/contacts/ContactForm.tsx

```diff
@@ -91,7 +91,7 @@ export default function ContactForm({
           window.history.replaceState(
             null,
             "",
-            `/administration/contacts/${response.id}`,
+            `/administration/contacts/${response.id}?contactsTitle=${response.first_name} ${response.last_name}`,
```

Comment:
This is really nitty but I just noticed that we are being inconsistent with our search params - the DataGrid uses `snake_case` but this is using `camelCase`.   
  
I don't have a strong preference for either, though I think we should keep them consistent. I'd lean towards keeping `snake_case` purely because I think it would be quite a bit more work to refactor as our many DataGrid search API routes are using those.